File: Assets/_Complete-Game/Scripts/Menu.js
	#pragma strict

	/*
	function Start () {
		
	}

	function Update () {
		
	}

	function Continue(){
	    
	}

	function Options(){
	    
	}

	*/
	function MenuPrincipal(){
		Application.LoadLevel("Menu");
	}

	function IniciarJogo(){
		Application.LoadLevel("_Complete-Game"); 
	}

	function Creditos(){
		Application.LoadLevel("Creditos");
	}

	function Sair(){
		Application.Quit();
	}

File: Assets/_Complete-Game/Scripts/Menu.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.SceneManagement;

public class Menu : MonoBehaviour {
	/*
	// Use this for initialization
	void Start () {
		
	}
	
	// Update is called once per frame
	void Update () {
		
	}

	*/

	public void MenuPrincipal(){
		SceneManager.LoadScene("Menu");
	}

	public void IniciarJogo(){
		SceneManager.LoadScene("_Complete-Game");
	}

	public void Creditos(){
		SceneManager.LoadScene("Creditos");
	}

	public void Sair(){
		Application.Quit();
	}
}
